### ISSUE TITLE
Rails 5.2 beta breaks javascript packs

### ISSUE THREAD

**Issue by Onumis** [rails/rails#31258]
### Steps to reproduce

- Create new project using `Rails 5.2.0.beta2`: `rails new ... --webpack=react`
- scaffold a controller and page
- add to view: `<%= javascript_pack_tag 'hello_react' %>`
- load `http://localhost:3000/`

### Expected behavior
Page to include `Hello React!`

### Actual behavior
Page does not include `Hello React!`, and JS console reports:

```
hello_react-5612ffe4e734573493a4.js:173
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' https:".

    at Object.<anonymous> (hello_react-5612ffe4e734573493a4.js:173)
    at __webpack_require__ (hello_react-5612ffe4e734573493a4.js:20)
    at hello_react-5612ffe4e734573493a4.js:63
    at hello_react-5612ffe4e734573493a4.js:66
```

### System configuration
**Rails version**: Rails 5.2.0.beta2

**Ruby version**: ruby 2.4.1p111

**Comment by Onumis** [comment]
From Gemfile.lock:

```
    webpacker (3.0.2)
      activesupport (>= 4.2)
      rack-proxy (>= 0.6.1)
      railties (>= 4.2)
```
let me know if you need more info.

**Comment by pixeltrix** [comment]
@Onumis this is a known issue - see: https://github.com/rails/rails/pull/31162#issuecomment-347154237

TL;DR you need to change `config.devtool` to `cheap-module-source-map` from `cheap-eval-source-map` - we'll be changing the default in webpacker itself before the final release of Rails 5.2.
